utils/npm/version.go

```diff
@@ -0,0 +1,66 @@
+package npmutils
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/utils/version"
+	"io"
+	"io/ioutil"
+	"os/exec"
+
+	gofrogcmd "github.com/jfrog/gofrog/io"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+func Version(executablePath string) (*version.Version, error) {
+	pipeReader, pipeWriter := io.Pipe()
+	defer pipeReader.Close()
+	defer pipeWriter.Close()
+	var npmError error
+
+	configListCmdConfig := createVersionCmdConfig(executablePath, pipeWriter)
+	go func() {
+		npmError = gofrogcmd.RunCmd(configListCmdConfig)
```

Comment:
I suggest we avoid using ```gofrogcmd.RunCmd``` when not necessary, but use go's default cmd API instead. The ```gofrog``` API is useful when the output needs to be parsed using a regexp, but as you may remember, it has its limitation. See other places the code where we use the default go API.